common/game_rules/000_wc_game_rules.txt

```diff
@@ -95,35 +95,35 @@ horde_invasion = {
 		desc = "RULE_HORDE_INVASION_STRENGTH_X0_5_DESC"
 	}
 }
-qiraji_invasion = {
-	name = "RULE_QIRAJI_INVASION_STRENGTH"
-	group = "RULE_GROUP_WARCRAFT"
-	option = {
-		name = default
-		text = "RULE_QIRAJI_INVASION_STRENGTH_DEFAULT"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_DEFAULT_DESC"
-	}
-	option = {
-		name = x1_5
-		text = "RULE_INVADERS_STRENGTH_X1_5"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_X1_5_DESC"
-	}
-	option = {
-		name = x2
-		text = "RULE_INVADERS_STRENGTH_X2"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_X2_DESC"
-	}
-	option = {
-		name = x0_25
-		text = "RULE_INVADERS_STRENGTH_X0_25"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_X0_25_DESC"
-	}
-	option = {
-		name = x0_5
-		text = "RULE_INVADERS_STRENGTH_X0_5"
-		desc = "RULE_QIRAJI_INVASION_STRENGTH_X0_5_DESC"
-	}
-}
+# qiraji_invasion = {
```

Comment:
Maybe replace it with `black_empire_invasion`? I still see you use `spawn_qiraji_troops_effect`.

Reply:
maybe old god freed army size?

Reply:
> maybe old god freed army size?

Just `old_gods_invasion`?

Reply:
But don't forget to unite it with `black_empire_invasion`.

Reply:
whadya mean unite?

Reply:
> whadya mean unite?

@sejtalk , `black_empire_invasion` can disable the Black Empire. You may unite it into one game rule that disables waking up Old Gods and regulates troops.